src/components/pages/HeadsUpDisplay/NextActivity.js

```diff
@@ -88,14 +50,15 @@ export const NextActivity = () => {
       12: "Matchup 4",
       13: "Big Final Reveal"
     };
-    setActivity(activityList[day+1]);
+    setActivity(activityList[day + 1]);
     return activity;
   };
 
+  // useEffect to call both functions on render
   useEffect(() => {
     nextDay();
     nextActivity();
   });
```

Comment:
I still think there should be some sort of dependency with eslint ignore here. wouldn't want this to an infinite loop.

Reply:
I have tried using every variation of dependancies that I could, some resulted in infinite loops and some caused the component to not re-render.  I am open to suggestions if you have any, but this is the only set up that functions properly.  I did run through the full cycle many times with no infinite loop, but if you see something I'm missing I will try it out.